dungeon/assetlist.json

```
@@ -0,0 +1,33 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "dungeon",
+  "assets": [
+    {
+      "description": "Dungeon native token",
+      "denom_units": [
```

```suggestion
      "denom_units": [
        {
          "denom": "udgn",
          "exponent": 6
        }
      ],
```

Please use this one. Don't use exponent 0.